share/p2p/shrexpush/pubsub.go

```diff
@@ -77,7 +81,7 @@ func (s *PubSub) AddValidator(validate Validator) error {
 // Subscribe provides a new Subscription for EDS notifications.
 func (s *PubSub) Subscribe() (*Subscription, error) {
 	if s.topic == nil {
-		return nil, fmt.Errorf("share/eds: topic is not started")
+		return nil, fmt.Errorf("share/shrexpush: topic is not started")
```

Comment:
nit, but this seems to be enough
```suggestion
		return nil, fmt.Errorf("shrexpush: topic is not started")
```